src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,30 +42,33 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              Todo #
+              {todo?.id}
             </div>
 
-            {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
             <button
               type="button"
+              aria-label="Mute volume"
               className="delete"
               data-cy="modal-close"
+              onClick={handelCloseModal}
             />
           </header>
 
           <div className="modal-card-body">
             <p className="block" data-cy="modal-title">
-              quis ut nam facilis et officia qui
+              {todo?.title}
```

Comment:
Why todo and user are not necessary. What will happen with your component if there will no data?